tests/job_progress/test_job_progress.py

```diff
@@ -331,54 +343,56 @@ def test_request_middleware_when_authenticated_passes_request_with_logged_in_use
 }
 
 
-class OrderUpdateTest(TestCase):
+class ExportUpdateTest(TestCase):
```

Comment:
I think it's good idea to keep it as TestCase Class instead of moving directly to the new pytest funtcions :+1: